rai/node/node.cpp

```diff
@@ -2806,9 +2802,31 @@ void rai::election::confirm_cutoff (MDB_txn * transaction_a)
 
 void rai::election::vote (std::shared_ptr<rai::vote> vote_a)
 {
-	node.network.republish_vote (last_vote, vote_a);
-	last_vote = std::chrono::steady_clock::now ();
+	// see republish_vote documentation for an explanation of these rules
 	rai::transaction transaction (node.store.environment, nullptr, true);
+	auto supply (node.ledger.supply (transaction));
```

Comment:
I think it's ideal to put the units here.  These are both raw?

Reply:
Yeah. Units aren't really important here though, as everything is in proportions.